### PR TITLE
Integrate select query with new processor

### LINKED ISSUE #3931: integrate new processor framework
- [x] integrate standalone select query with new processor
    - [x] integrate numbers table with new processor
    - [x] integrate systems table with new processor
    - [x] #4345
- [x] #4471
- [x] #4799

### PR DESCRIPTION
**Pull request by zhang2014**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Integrate select query with new processor

## Changelog

- New Feature

## Related Issues

part #3931

## Test Plan

Unit Tests

Stateless Tests

### CODE REVIEW

**Comment by zhang2014** [comment]
Implemented:
- [ ] numbers table function with new processor
- [ ] system.one table with new processor
- [ ] where clause with new processor
- [ ] group by clause with new processor
- [ ] having clause with new processor
- [ ] order by clause with new processor
- [ ] limit clause with new processor
- [ ] limit by clause with new processor
- [ ] projection and expressions with new processor

File: query/src/sessions/session_settings.rs [hunk 1]
```diff
@@ -74,66 +74,74 @@ impl Settings {
         let values = vec![
             // max_block_size
             SettingValue {
-                default_value:DataValue::UInt64(10000),
+                default_value: DataValue::UInt64(10000),
```
Comment (BohuTANG) [inline query/src/sessions/session_settings.rs:77]: :+1: 
Not sure why `make lint` not work for it :crying_cat_face:

Comment (zhang2014) [reply query/src/sessions/session_settings.rs:77]: Clion can format this code. But I cannot find relevant configuration from https://rust-lang.github.io/rustfmt :(

Comment (zhang2014) [reply query/src/sessions/session_settings.rs:77]: It seems to triggered some bugs : ( 

https://rust-lang.github.io/rustfmt/?version=v1.4.38&search=col#space_after_colon

**Pull request merged**